### PR TITLE
[Issue #6843] Remove six library

### PR DESCRIPTION
**Pull request by burner761**
remove all uses of the six library in the project.

### CODE REVIEW

**Review by afabiani — APPROVED** [review]

**Comment by afabiani** [comment]
@burner761 can you please take a look at the new LGTM issues introduced with this PR?

File: geonode/monitoring/collector.py [hunk 1]
```diff
@@ -452,7 +451,7 @@ def _key(v):
                                   'label': label,
                                   'samples_count': samples,
                                   'value_raw': value or 0,
-                                  'value_num': value if isinstance(value, integer_types + (float, Decimal,)) else None})
+                                  'value_num': value if isinstance(value, int + (float, Decimal,)) else None})
```
Comment (mtnorthcott) [inline geonode/monitoring/collector.py:454]: `integer_types` is a collection/sequence so `integer_types + (float, Decimal)` extends the sequence. `int` is not a sequence and does not support the `+` operation on the `tuple` type. Please fix this to simply contain `int` within the `tuple`.

**Review by mtnorthcott — CHANGES_REQUESTED** [review]
`six` can still be removed from the following:
* geonode/social/tests.py
* geonode/geoapps/api/serializers.py
* geonode/geoserver/tasks.py
* requirements.txt

**Review by afabiani — CHANGES_REQUESTED** [review]
There are some conflicts, moreover see if it is possible to solve the new LGTM issues.

**Review by afabiani — APPROVED** [review]

**Pull request merged**